zookeeper/Dockerfile

```diff
@@ -49,14 +71,56 @@ COPY shared/log4shell.sh /bin
 RUN /bin/log4shell.sh /stackable/apache-zookeeper-${PRODUCT}-bin
 
 # Ensure no vulnerable files are left over
-# This will currently report vulnerable files being present, as it also alerts on
-# SocketNode.class, which we do not remove with our scripts.
-# Further investigation will be needed whether this should also be removed.
+# This will currently report vulnerable files being present, as it also alerts
+# on SocketNode.class, which we do not remove with our scripts. Further
+# investigation will be needed whether this should also be removed.
 COPY shared/log4shell_1.6.1-log4shell_Linux_x86_64 /bin/log4shell_scanner_x86_64
 COPY shared/log4shell_1.6.1-log4shell_Linux_aarch64 /bin/log4shell_scanner_aarch64
 COPY shared/log4shell_scanner /bin/log4shell_scanner
 RUN /bin/log4shell_scanner s /stackable/apache-zookeeper-${PRODUCT}-bin
 # ===
 
+# END ZOOKEEPER BUILDER
+# ==============================================================================
+
+# ==============================================================================
+# START FINAL ZOOKEEPER IMAGE
+
+FROM stackable/image/java-base
+
+ARG PRODUCT
+ARG RELEASE
+
+LABEL name="Apache ZooKeeper" \
+      maintainer="info@stackable.tech" \
+      vendor="Stackable GmbH" \
+      version="${PRODUCT}" \
+      release="${RELEASE}" \
+      summary="The Stackable image for Apache ZooKeeper." \
+      description="This image is deployed by the Stackable Operator for Apache ZooKeeper."
+
+RUN microdnf update && \
+    microdnf install tar gzip zip && \
+    microdnf clean all && \
+    rm -rf /var/cache/yum
+
+# COPY --chown=stackable:stackable zookeeper/stackable /stackable
+COPY zookeeper/licenses /licenses
+
+USER stackable
+WORKDIR /stackable
+
+# Copy over the complete Zookeeper binary folder
+COPY --link --chown=stackable:stackable --from=builder /stackable/apache-zookeeper-${PRODUCT}-bin /stackable/apache-zookeeper-${PRODUCT}-bin/
+COPY --link --chown=stackable:stackable --from=builder /stackable/jmx /stackable/jmx/
+
+# Add link pointing from /stackable/zookeeper to
+# /stackable/apache-zookeeper-${PRODUCT}-bin/ to preserve folder name
+# with version.
+RUN ln -s /stackable/apache-zookeeper-${PRODUCT}-bin/ /stackable/zookeeper
+
```

Comment:
While you're at it: Can you include a PATH and package manifest as well?
See Hive for an example.

Reply:
https://github.com/stackabletech/docker-images/pull/638/commits/c21d8e977d69f759ece7dc7bcf18c1f876775f75 and https://github.com/stackabletech/docker-images/pull/638/commits/982d061399fef7c8a4e181d6e9c5f5dd40c33a1b